enclave-manager/web/src/client/enclaveManager/KurtosisClientContext.tsx

```diff
@@ -7,6 +7,8 @@ import { KURTOSIS_CLOUD_EM_PAGE, KURTOSIS_CLOUD_EM_URL } from "../constants";
 import { AuthenticatedKurtosisClient } from "./AuthenticatedKurtosisClient";
 import { KurtosisClient } from "./KurtosisClient";
 import { LocalKurtosisClient } from "./LocalKurtosisClient";
+import React from "react";
+import { EnclaveInfo } from "enclave-manager-sdk/build/engine_service_pb";
```

Comment:
Don't think anything has changed in this file to warrant this. In this part of the codebase we use a code formatter called `prettier` - you can either have it automatically run on save by your ide, or run `yarn prettier:fix` to run across all of the code.

Reply:
nice tip! just added it to my IDE